release-guide.md

```diff
@@ -0,0 +1,104 @@
+# SIARD Suite Release Testing Guide
+
+## Motivation
+
+The development of SIARD Suite is financed by the Swiss Federal Archives (SFA), which is one of the main stakeholders of the project. The SFA is also one of the main users of the application. There are many restrictions on the workstations that can be used by SFA staff:
+
+- Pre-installed Java 8 (+JavaFX) with no possibility to switch to another JRE.
+- Inability to download and run any form of executable binary or script (e.g. `SIARD-suite.bat`).
+
+The only way SFA staff can run the application is with the following cli command
+
+```bash
+java -jar lib/siard-suite.jar
+```
+
+Due to other constraints, it has happened on a regular basis that the application runs fine when using the native installers or start scripts (with bundled jre), but not with the above command, without anyone noticing until an SFA staff member tries to run the application without success.
```

Comment:
Due to other constraints, the application will often run fine when using native installers or start scripts (with bundled JRE), but not with the above command, without anyone noticing until an SFA staff member tries to run the application without success.